src/sagemaker/huggingface/training_compiler/config.py

```diff
@@ -102,6 +102,17 @@ def validate(cls, estimator):
 
         super(TrainingCompilerConfig, cls).validate(estimator)
 
+        if estimator.pytorch_version:
+            if (Version(estimator.pytorch_version) in SpecifierSet("< 1.9")) or (
+                Version(estimator.pytorch_version) in SpecifierSet("> 1.11")
+            ):
+                error_helper_string = (
+                    "SageMaker Training Compiler is only supported "
+                    "with HuggingFace PyTorch 1.9-1.11. "
+                    "Received pytorch_version={} which is unsupported."
+                )
+                raise ValueError(error_helper_string.format(estimator.pytorch_version))
```

Comment:
nit: can you please variables same as in tensorflow for max and min version. 
Otherwise LGTM

Reply:
Thanks for the comment. Since we won't change this version support in the future for HuggingFace training compiler estimator support which has been restrict to 1.9-1.11, it is not a big deal for us to variables this now.